docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -86,6 +87,14 @@ $ cmake ..
 $ make
 ```
 
+### Configuring OCaml for Windows
+Before building, there are a few OCaml configuration steps:
+
+1. Add `ocaml-env.exe` to the `PATH` environment variable. The default path of `ocaml-env.exe` is  `C:\OCaml64\usr\local\bin\ocaml-env.exe`.
+2. Open `cmd.exe` (or another shell) and run `ocaml-env exec -- cmd.exe`. This is create a OCaml environment in the shell.
+3. Install `ocamlbuild` by running `opam install ocamlbuild`.
+4. Exit out of the shell.
```

Comment:
Is this a one time thing or every time you build?

Reply:
Yes, it's a one time thing. I'll make that more clear in the docs.